src/renderer/gl_engine/tvgGlGpuBuffer.cpp

```diff
@@ -102,37 +102,56 @@ uint32_t GlStageBuffer::push(void *data, uint32_t size, bool alignGpuOffset)
     return offset;
 }
 
+uint32_t GlStageBuffer::pushIndex(void *data, uint32_t size)
+{
+    uint32_t offset = mIndexBuffer.count;
+
+    if (this->mIndexBuffer.reserved - this->mIndexBuffer.count < size) {
```

Comment:
this (109 ~ 111) looks not necessary. 
At reserve you want to use? `reserve()` instead of `grow()`

Reply:
I just want to make sure that we need to resize the container